flutter/lib/components/todo_list.dart

```diff
@@ -4,16 +4,15 @@ import 'package:realm/realm.dart';
 import 'todo_item.dart';
 import 'package:flutter_todo/realm/schemas.dart';
 
-const _animationDuration = Duration(milliseconds: 300);
-
 class TodoViewModel {
   final String id;
   final bool isComplete;
```

Comment:
The logic written behind onChanged could be a method or setter SetState in TodoViewModel with a nullable argument. On this way the View will be kept simple as possible.
``` dart
  return Checkbox(
      checkColor: Colors.white,
      fillColor: MaterialStateProperty.resolveWith(getColor),
      value: viewModel.isComplete,
      onChanged: (bool? value) { //This could be moved to TodoViewModel.
        realm.write(() {
          viewModel.todo.isComplete = value ?? false;
        });
      },
    );
```